torch/_dynamo/variables/builtin.py

```diff
@@ -1431,6 +1432,12 @@ def _unimplemented():
         if isinstance(left, TensorVariable):
             from .builder import wrap_fx_proxy_cls
 
+            if op is operator.is_ and isinstance(right, TensorVariable):
+                return ConstantVariable.create(
+                    id(extract_fake_example_value(left.as_proxy().node))
+                    == id(extract_fake_example_value(right.as_proxy().node))
```

Comment:
why not `is` ?

Reply:
Yeah, I guess we could use just use is? But it is the same thing for Tensor.